pkg/commands/artifact/run.go

```diff
@@ -413,6 +413,19 @@ func disabledAnalyzers(opts flag.Options) []analyzer.Type {
 		analyzers = append(analyzers, analyzer.TypeLanguages...)
 	}
 
+	// Do not perform vuln scanning when it is not specified.
+	// We need these analyzers to find packages for `cyclonedx` format. Vulns will be excluded in `Report` package
+	if !slices.Contains(opts.SecurityChecks, types.SecurityCheckVulnerability) && opts.Format != "cyclonedx" {
```

Comment:
I think it causes a problem with SPDX. Also, there are `spdx-json` and `github`.